WordPress/WordPressTest/MediaEditorTests.swift

```diff
@@ -0,0 +1,129 @@
+import XCTest
+import TOCropViewController
+import Nimble
+
+@testable import WordPress
+
+class MediaEditorTests: XCTestCase {
+
+    private var cropViewControllerMock: TOCropViewControllerMock!
+    private var mediaEditor: MediaEditor!
+
+    override func setUp() {
+        super.setUp()
+        cropViewControllerMock = TOCropViewControllerMock()
+        mediaEditor = MediaEditor(cropViewControllerFactory: cropViewControllerMockFactory)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        cropViewControllerMock = nil
+        mediaEditor = nil
+    }
+
+    func testGiveTheImageToCropViewController() {
+        let image = UIImage()
+        var imageToCrop: UIImage?
+        let mediaEditor = MediaEditor(cropViewControllerFactory: { image in
+            imageToCrop = image
+            return self.cropViewControllerMock
+        })
+
+        mediaEditor.edit(image, onFinishEditing: { _ in })
+
+        expect(imageToCrop).to(equal(image))
+    }
+
+    func testReturnsTheCroppedImage() {
+        let originalImage = UIImage()
+        let croppedImage = UIImage()
+        var returnedImage: UIImage?
+
+        mediaEditor.edit(originalImage, onFinishEditing: { croppedImage in
+        returnedImage = croppedImage
+
```

Comment:
Extra line. Did you originally have `cropViewControllerMock.croppedImage = croppedImage` in the onFinishEditing?

Reply:
Fixed!

> Did you originally have `cropViewControllerMock.croppedImage = croppedImage` in the onFinishEditing?
It was outside `onFinishEditing`. :)